taxonomies/food/ingredients.txt

```diff
@@ -90846,6 +90854,7 @@ nl:sojabereiding
 pt:preparado à base de soja
 sv:sojabönsberedning
 allergens:en: en:soybeans
+nova:en:4
```

Comment:
soy preparation could be anything, it should not be NOVA 4.

Reply:
so we can not set anything. Or do we go for the worst case?

Reply:
we don't go for the worse case. In most cases, we should have sub ingredients for things like "XYZ preparation", and those can trigger NOVA 3 or 4.